lib/ReactViews/Credits/Credit.type.ts

```diff
@@ -0,0 +1,4 @@
+export interface ICredit {
+  url: string;
+  text: string;
+}
```

Comment:
I think that the `Checkbox/types.ts` and `Styled.types.ts` are the only files of this type (and guess who added them 🤣). It won't be bad to separate types from components when they are used in multiple places. In the end, this will be in traits in a future version of terria since it is used in config.

Reply:
OK agreed